src/main/java/com/homeadvisor/kafdrop/controller/ClusterController.java

```diff
@@ -112,4 +118,10 @@ private String brokerNotFound(Model model)
       public List<BrokerVO> brokers;
       public List<TopicVO> topics;
    }
+
+   @ResponseStatus(HttpStatus.OK)
+   @RequestMapping("/health_check")
+   public void healthCheck()
```

Comment:
Spring Boot already provides a health status endpoint.  What is this endpoint needed for?